src/main/java/edu/wright/airviewer2/AIRViewerController.java

```diff
@@ -392,6 +395,20 @@ public void handle(ActionEvent e) {
 		refreshUserInterface();
 		return model;
 	}
+	
+	 /**
+     * Initializes about menu function
+     */
+
```

Comment:
it's better to format the comments section to make it more readable and structed.

Reply:
changed accordingly